src/main/java/racingcar/Winners.java

```diff
@@ -0,0 +1,30 @@
+package racingcar;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Winners {
+    private static final int MIN_POSITION = 0;
```

Comment:
```suggestion
    private static final Position MIN_POSITION = 0;
```

> 규칙 3: 모든 원시값과 문자열을 포장한다.


이름은 `Position`인데 만들어주신 객체를 활용안한 이유가 있으신가요?

Reply:
뜨끔.. 수정했습니다.